.version

```diff
@@ -1 +1 @@
-1.125.2
+1.13.12
```

Comment:
This is confusing because the .version in the main branch right now is already 1.13.12 (not 1.125.2).